sources/pvmp3dec/src/pvmp3_decode_huff_cw.cpp

```diff
@@ -139,7 +139,9 @@
 
 uint16 pvmp3_decode_huff_cw_tab0(tmp3Bits *pMainData)
 {
+#if (CC_PLATFORM != CC_PLATFORM_NX && CC_PLATFORM != CC_PLATFORM_NX_WINDOWS)
     OSCL_UNUSED_ARG(pMainData);
+#endif
```

Comment:
Why need to modify it?

Reply:
this macro defined in a temp .h file

when project in chinese dir, this auto generate command line will no use on nx :
![image](https://user-images.githubusercontent.com/9692838/169976089-72f06855-fdea-43e2-ac06-dba0c091c7b6.png)

there ara no problem if use english path.